UseWebThreeJsonRpc.cmake

```diff
@@ -1,19 +1,26 @@
 function(eth_apply TARGET REQUIRED)
 	eth_use(${TARGET} ${REQUIRED} EthCore ServerJsonRpc)
-	set(W3_DIR         "${PROJECT_SOURCE_DIR}/../webthree" 		CACHE PATH "The path to the webthree directory")
-	set(W3_BUILD_DIR_NAME  "build"                            	CACHE STRING "The name of the build directory in web3")
-	set(W3_BUILD_DIR   "${W3_DIR}/${W3_BUILD_DIR_NAME}")
-	set(CMAKE_LIBRARY_PATH ${W3_BUILD_DIR}/src;${CMAKE_LIBRARY_PATH})
-	include_directories(${W3_DIR})
-	find_library(ETH_WEBTHREE_LIBRARY NAMES webthree PATH_SUFFIXES "libwebthree" "webthree" "libwebthree/Release" )
-	if (NOT ETH_WEBTHREE_LIBRARY)
-		if (${REQUIRED} STREQUAL "REQUIRED")
-			message( FATAL_ERROR "Webthree library not found" )
+
+	if (DEFINED ethereum_SOURCE_DIR)
```

Comment:
`ethereum_SOURCE_DIR was there since ages. It is set by `project(ethereum)`. `W3_DIR` is The Yet Another Variable™.

And `set(W3_DIR ...)` will override it even if set in parent scope. With `CACHE` is even worse. The first build is different than the second one (tested).